src/app/admin/usersTable/UsersTable.tsx

```diff
@@ -230,6 +231,26 @@ const UsersTable: React.FC = () => {
         void fetchAndDisplayUserData();
     }, [fetchAndDisplayUserData]);
 
+    useEffect(() => {
+        void fetchAndDisplayUserData();
```

Comment:
why do we need to fetch here? I thought this useEffect should just run once at the beginning to try and subscribe to the db changes?